lib/src/definitions.ts

```diff
@@ -63,6 +63,7 @@ export interface PathParam {
   name: string;
   description?: string;
   type: Type;
+  examples?: Record<string, string>;
```

Comment:
```suggestion
  examples?: Example[];
```

The convention we currently use is a sorted array. There is consideration being put into changing this convention to use `Record`s. For now, use arrays for consistency.